src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -338,11 +338,23 @@ internal NamespaceOrTypeOrAliasSymbolWithAnnotations BindNamespaceOrTypeOrAliasS
             {
                 case SyntaxKind.NullableType:
                     {
-                        TypeSyntax typeArgumentSyntax = ((NullableTypeSyntax)syntax).ElementType;
+                        var nullableSyntax = (NullableTypeSyntax)syntax;
+                        TypeSyntax typeArgumentSyntax = nullableSyntax.ElementType;
                         TypeSymbolWithAnnotations typeArgument = BindType(typeArgumentSyntax, diagnostics, basesBeingResolved);
                         TypeSymbolWithAnnotations constructedType;
                         if (Compilation.IsFeatureEnabled(MessageID.IDS_FeatureStaticNullChecking))
                         {
+                            if (InExecutableBinder)
+                            {
+                                // Inside a method body or other executable code, we can afford to pull on NonNullTypes symbol or question IsValueType without causing cycles.
```

Comment:
>afford to [](start = 89, length = 9)

Nit: delete.

Reply:
I'll fix in next PR. Thanks

---
In reply to: [208040660](https://github.com/dotnet/roslyn/pull/28842#discussion_r208040660) [](ancestors = 208040660)